app/src/main/java/com/example/habitsmasher/UserRegistrationFragment.java

```diff
@@ -257,4 +307,38 @@ public void onStop() {
         if (supportActionBar != null)
             supportActionBar.show();
     }
+
+    /**
+     * Not touching this when refactoring until images are fully implemented for habit events
+     * Reference: https://stackoverflow.com/questions/10165302/dialog-to-pick-image-from-gallery-or-from-camera
+     * Override onActivityResult to handle when user has selected image
+     * @param requestCode
+     * @param resultCode
+     * @param imageReturnedIntent
+     */
+    @Override
+    public void onActivityResult(int requestCode, int resultCode, Intent imageReturnedIntent) {
+        super.onActivityResult(requestCode, resultCode, imageReturnedIntent);
+        switch(requestCode) {
+            case 0:
+            case 1:
+                if (resultCode == RESULT_OK) {
+                    // Set selected picture
+                    _selectedImage = imageReturnedIntent.getData();
+                    _profilePictureView.setImageURI(_selectedImage);
+                }
+                break;
+        }
+    }
```

Comment:
I notice this method has appeared (at least) twice now. Is there any way we can abstract this to a common class, either an existing one or one that's brand new.